### DISCUSSION TITLE
SetCell with NewColumnType

### DISCUSSION THREAD

**Discussion by jonhall** [lvjr/tabularray#537] [Q&A]
From the manual I can write, for instance:

> `\NewColumnRowType { ! } [1] { |[text={#1}] }`

to create a new column type which can be used as a column specifier in a tblr, but not in the column specifier in SetCell. Is that correct?

**Comment by lvjr** [comment]
Yes, it is correct. I ever considered the feature of setting cell borders in `\SetCell` command, but couldn't design a clean interface for it.

Update: I have created issue https://github.com/lvjr/tabularray/issues/548 .

**Comment by jonhall** [comment]
Many thanks, lvjr, and thanks again for tabularray.
